plugins/modules/sda_fabric_devices_workflow_manager.py

```diff
@@ -2344,14 +2362,41 @@ def get_device_params(self, fabric_id, network_id, device_details, config_index)
                     return self.check_return_status()
 
             try:
-                local_autonomous_system_number = int(local_autonomous_system_number)
-                if not 1 <= local_autonomous_system_number <= 4294967295:
-                    self.msg = (
-                        "The 'local_autonomous_system_number' should be from 1 to 4294967295 for the device '{ip}'."
-                        .format(ip=device_ip)
-                    )
-                    self.status = "failed"
-                    return self.check_return_status()
+                if "." in str(local_autonomous_system_number):
```

Comment:
```
str_asn = str(local_autonomous_system_number)
if "." in str_asn:
    parts = str_asn.split(".")
```

Reply:
Can we write as a function?
def validate_local_autonomous_system_number(self, local_autonomous_system_number):
OR
def validate_local_autonomous_system_number(self, local_autonomous_system_number, device_ip):